File: LNPG_01_TIPOS_E_VARIAVEIS/ATIVIDADE_1/C/main.c
#include <stdio.h>

double soma(n1, n2)
{
  return n1 + n2;
};
double sub(n1, n2)
{
  return n1 - n2;
};
double mult(n1, n2)
{
  return n1 * n2;
};
double div(n1, n2)
{
  return n1 / n2;
};

int main()
{
  double n1;
  double n2;

  printf("Digite o primeiro número: ");
  scanf("%lf", &n1);
  printf("Digite o segundo número: ");
  scanf("%lf", &n2);

  printf("A soma dos números é: %.2f\n", soma(n1, n2));
  printf("A subtração dos números é: %.2f\n", sub(n1, n2));
  printf("A multiplicação dos números é: %.2f\n", mult(n1, n2));
  printf("A divisão dos números é: %.2f\n", div(n1, n2));

  return 0;
}

File: LNPG_01_TIPOS_E_VARIAVEIS/ATIVIDADE_1/HASKELL/main.hs
soma :: Float -> Float -> Float
soma x y = x + y
sub :: Float -> Float -> Float
sub x y = x - y
mult :: Float -> Float -> Float
mult x y = x * y
divi :: Float -> Float -> Float
divi x y = x / y

main::IO()
main = do 
  putStrLn "Digite o primeiro numero: "
  input1 <- getLine
  let n1 = read input1 :: Float
  putStrLn "Digite o segundo numero: "
  input2 <- getLine
  let n2 = read input2 :: Float
  putStrLn ("A soma dos numeros e " ++ show (soma n1 n2))
  putStrLn ("A subtracao dos numeros e " ++ show (sub n1 n2))
  putStrLn ("A multiplicacao dos numeros e " ++ show (mult n1 n2))
  putStrLn ("A divisao dos numeros e " ++ show (divi n1 n2))